packages/blade/src/components/BottomSheet/_decisions/decisions.md

```diff
@@ -0,0 +1,245 @@
+# Bottom Sheet
+
+Bottom sheet is a container that presents additional information to a user in an overlay.
+
+This document outlines the API of `BottomSheet` component.
+
+<img src="./bottom-sheet-thumbnail.png" width="380" alt="Thumbnail of bottom sheet" />
+
+- [Bottom Sheet](#bottom-sheet)
+  - [Design](#design)
+  - [Anatomy](#anatomy)
+  - [API](#api)
+    - [`BottomSheet`](#bottomsheet)
+    - [`BottomSheetBody`](#bottomsheetbody)
+    - [`BottomSheetHeader` API](#bottomsheetheader-api)
+    - [`BottomSheetFooter` API](#bottomsheetfooter-api)
+  - [Composition with Dropdown](#composition-with-dropdown)
+  - [Accessibility](#accessibility)
+  - [Open questions](#open-questions)
+  - [Alternative APIs](#alternative-apis)
+  - [References](#references)
+
+## Design
+
+- [Figma - BottomSheet](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=26477%3A578228&t=1RMqjs99P33Udj0C-0)
+
+## Anatomy
+
+<img src="./bottomsheet-anatomy.png" alt="BottomSheet Anatomy" width="100%" />
+
+
+Components:
+
+- BottomSheet
+  - BottomSheetHeader
+  - BottomSheetBody
+  - BottomSheetFooter
+
+## API
+
```

Comment:
We need a small discussion around this, wether to keep isOpen state controlled or uncontrolled, let's discuss this tommorow on standup along with the Saurabh's discussion about the Dropdown.

Reply:
So right now it behaves like controlled. 

> What if someone passed isOpen as true and then drags it down to shut it
it will trigger the onDismiss function which then user will set the setIsOpen(false)

So the usage will be like: 

```
const [isOpen, setOpen] = React.useState(false);

<BottomSheet isOpen={isOpen} onDismiss={() => setIsOpen(false)} />
```